stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -175,13 +175,45 @@ mul!(C::AbstractMatrix, A::AbstractVecOrMat, B::AbstractVecOrMat) = generic_matm
     rmul!(A, B)
 
 Calculate the matrix-matrix product ``AB``, overwriting `A`, and return the result.
+
+# Examples
+```jldoctest
+julia> using LinearAlgebra;
```

Comment:
Semicolon snuck in? :)

Reply:
I also don't think we have `using LinearAlgebra` in any other doctests?